spec/schema_spec.rb

```diff
@@ -185,7 +185,7 @@ def self.name
         self.description = "An employee of the organization"
 
         attribute :first_name, :string, description: "The employee's first name"
-
+        attribute :hidden_attribute, :string, schema: false
```

Comment:
I know we have the diff spec, but should probably be one here too that flips the boolean and asserts it's now present.

Reply:
Added spec